nucliadb/nucliadb/ingest/consumer/pull.py

```diff
@@ -192,6 +193,15 @@ async def finalize(self):
             await self.nc.close()
 
     async def subscription_worker(self, msg: Msg):
+        if len(NODES) == 0:
```

Comment:
Could you add a comment here. It's not obvious that we are waiting to get node metadata for chitchat with a 1s sleep.

Furthemore, why adding an extra ack_wait (10s) and wait only 1s? The remaining 9s wouldn't be enough?

Reply:
yeah, this is silly. It should be a `while` and wait for it to become available.

The remaining 9s should be enough, however, the `in_progress` call resets the AckWait setting so it's like a new message until we're ready to work on it.